docs/src/main/paradox/scala/http/routing-dsl/source-streaming-support.md

```diff
@@ -87,42 +85,42 @@ in case one element in front of the stream takes a long time to marshall, yet ot
 
 ## Consuming JSON Streaming uploads
 
-Sometimes the client may be sending a streaming request, for example an embedded device initiated a connection with
+Sometimes a client sends a streaming request. For example, an embedded device initiated a connection with
 the server and is feeding it with one line of measurement data.
 
-In this example, we want to consume this data in a streaming fashion from the request entity, and also apply
-back-pressure to the underlying TCP connection, if the server can not cope with the rate of incoming data (back-pressure
-will be applied automatically thanks to using Akka HTTP/Streams).
+In this example, we want to consume this data in a streaming fashion from the request entity and also apply
+back pressure to the underlying TCP connection should the server be unable to cope with the rate of incoming data. Back pressure
+is automatically applied thanks to Akka HTTP's Streams library.
```

Comment:
I would say it's thanks Akka HTTP, and more precisely the Streams library.
Actually, Streams is not an Akka HTTP library, but an Akka one